src/configure/templates/nodejs.yml

```diff
@@ -6,6 +6,13 @@
 trigger:
 - {{ sourceRepository.branch }}
 
+variables:
+  # Azure Resource Manager connection
+  azureSubscription: '{{ targetResource.serviceConnectionId }}'
```

Comment:
this will still mean that service connection id is hard-coded in yaml. Should we set this as a variable while creating pipeline - which basically means a separate `variables` field in build definition payload.

Reply:
As discussed offline, leaving this as it is right now.